owa/app/js/home/home.html

```diff
@@ -1,3 +1,6 @@
+<div>
+    <a class="button" href="#/mapping"><i class="icon-plus"></i>&#160;Create Mapping</a>
```

Comment:
@dkayiwa I will do it once [this PR ](https://github.com/openmrs/openmrs-module-metadatamapping/pull/38/files) is merged. Is that ok ? :)